### DISCUSSION TITLE
Override table name for bulk actions

### DISCUSSION THREAD

**Discussion by Ceepster14** [rappasoft/laravel-livewire-tables#230] [Feature Requests]
Currently the table name for bulk actions is hard-coded i.e. "You have selected <strong>:count</strong> **users**,"

Would it be better to allow the table name to be set as a property in the component to avoid having to modify the published view?

**Comment by rappasoft** [comment]
I plan on adding a base translation file soon, so you'd be able to edit the English version of that.

**Reply by rappasoft** [reply]
Oh I just realized you meant the users part, yes that's an oversight on my end and I'll get that fixed.

**Reply by rappasoft** [reply]
Ok pull 1.2.2, I removed users and changed to generic 'rows' for now. You'll have the ability to customize that text when I finish localizing the package.

**Reply by Ceepster14** [reply]
Thanks! Great package by the way - looking forward to using it in anger!
